src/gasman.c

```diff
@@ -915,6 +915,10 @@ Bag NextBagRestoring( UInt type, UInt flags, UInt size )
   header->flags = flags;
   header->size = size;
   header->link = NextMptrRestoring;
+#if SIZEOF_VOID_P == 4
```

Comment:
Are we obsoleting SYS_IS_64_BIT in favour of checking SIZEOF_VOID_P directly? No objection to doing so, but we
should probably do it globally if so

Reply:
I was continuing to use SIZEOF_VOID_P, which was already being used to check this reserved value.

Reply:
I guess it is my "fault" that `gasman.h` uses `SIZEOF_VOID_P` and not `SYS_IS_64_BIT`; but in my defense (if that's necessary :-), that was because the "old" code, before I introduced `BagHeader`, also checked `SIZEOF_VOID_P` to set `USE_NEWSHAPE` -- perhaps it was added before `SYS_IS_64_BIT` existed?

Anyway, I have a slightly preference to use `SYS_IS_64_BIT`, *but* I am fine with this part of the PR either way; we can change `SIZEOF_VOID_P` checks to use `SYS_IS_64_BIT` later on if we want to -- or also not do it, I think the code is fine either way (if we ever get 128 bit systems, we'll have to go through it all anyway ;-).